src/Components/WhatWeDo/WhatWeDo.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import './WhatWeDo.css';
+import icon1 from './icons/icon1.png';
+import icon2 from './icons/icon2.png';
+import icon3 from './icons/icon3.png';
+
+export default function WhatWeDo(props) {
+  const title1 = '+ activities',
+    desc1 = 'Enjoy a variety of activities throughout the north!';
+  const title2 = 'Explore Cities',
+    desc2 = 'Every City has its own stuff, get to know them more with us';
+  const title3 = 'Have tour guide',
+    desc3 = 'First time in North? don’t worry you can have a tour guide';
+
+  return (
+    <div className="container">
+      <div className="part">
+        <img src={icon1} className="iconStyle" alt="icon" />
```

Comment:
if I was doing this, I will create a small function that takes an object as an arg or props and returns these elements and have an array of objects for these instead of title1 and title2, ...etc